libs/datasets/new_cases_and_deaths.py

```diff
@@ -110,10 +113,12 @@ def spread_first_reported_value_after_stall(
     num_days_worth_of_cases[zeros_to_replace] = None
 
     # Don't spread on first case
-    is_after_first_case = series.index > series.first_valid_index()
+    first_case = series[series.gt(0)].index[0]
+    is_after_first_case = series.index > first_case
```

Comment:
This and the change below fix the leading zeros issue I think...

Reply:
mmm yes makes sense.